tests/acceptance/01_vars/02_functions/execresult_4k.cf

```diff
@@ -0,0 +1,43 @@
+#######################################################
+#
+# Test returnszero()
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle agent init
+{
+}
+
+bundle agent test
+{
+  vars:
+    any::
+      "cwd" string => dirname("$(this.promise_filename)");
+    !windows::
+      "cat_4k" string => "cat $(cwd)$(const.dirsep)text_xform.cf.txt";
```

Comment:
Should probably use ``$(G.cat)`` instead of ``cat``. That also saves you from the Windows specific section below, since it will locate the Windows MSYS tools correctly.